CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [0.3.0](https://github.com/everypolitician/field_serializer/compare/v0.2.0...v0.3.0) - 2016-12-02
+
+### Added
+
+- There is now a proper README with some basic documentation.
+
+### Fixed
+
+- Inheritence now works as expected. Previously subclasses weren't getting the
```

Comment:
Good catch! That one always trips me up. Fixed in https://github.com/everypolitician/field_serializer/pull/13/commits/3b43fe27cf4560783617c6a25e4274ec2f9a27cd